src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -8,6 +8,14 @@ description: "Learn more about Sentry’s integration platform and how it provid
 
 Sentry’s integration platform provides a way for external services to interact with Sentry using [webhooks](/product/integrations/integration-platform/webhooks/), [UI components](/product/integrations/integration-platform/ui-components/), and the [REST API](/api). Integrations using this platform are first-class actors within Sentry.
 
+## Quick Start
+
+If you're new to the integration platform, it may help to get started with an example project to demo the different features and data avaiable your application. To help out, we've built a [reference implementation](https://github.com/getsentry/integration-platform-example) which you can use as a starting point.
```

Comment:
This first sentence is really long. I'll split it into three like: 
```
If you're new to the integration platform, it may help to get started with an example project. To help out, we've built a reference implementation which you can use as a starting point. It demos the different features and data available your application. 
```
also `avaiable` -> `available`.